neo4j-hot-backup@.service

```diff
@@ -1,25 +1,26 @@
 [Unit]
 Description=Neo4j hot backup
-After=neo4j@%i.service
-Requires=neo4j@%i.service
+After=neo4j@*.service
 
 [Service]
 Environment="DOCKER_APP_VERSION=latest"
 TimeoutStartSec=0
 KillMode=none
 
-ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
-ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
-ExecStartPre=/bin/bash -c '/usr/bin/docker exec $(docker ps | grep ft-neo4j | col | cut -f1) ./bin/neo4j-backup --host=127.0.0.1 --to=/data/backup -verify false'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=^/%p-%i_)" > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=^/%p-%i_)" > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker exec $(docker ps | grep neo4j | col | cut -f1) ./bin/neo4j-backup --host=127.0.0.1 --to=/data/backup -verify false'
 
 ExecStart=/bin/bash -c '\
 	/usr/bin/docker run --rm --name %p-%i_$(uuidgen) \
 	--env AWS_ACCESS_KEY_ID=$(/usr/bin/etcdctl get /ft/_credentials/aws/aws_access_key_id) \
 	--env AWS_SECRET_ACCESS_KEY=$(/usr/bin/etcdctl get /ft/_credentials/aws/aws_secret_access_key) \
 	--env AWS_BUCKET_NAME=com.ft.coco-neo4j-backup \
 	--env S3_DIR=$(/usr/bin/etcdctl get /ft/config/environment_tag) \
-	-v /vol/neo4j-%i/backup:/backup \
+	-v /vol/neo4j/data/backup:/backup \
```

Comment:
We mount to /vol/neo4j as opposed to /vol, but essentially yes.  Since people know where to look for data, there didn't seem an overriding reason to change it.